apps/judicial-system/web/src/routes/Shared/Cases/MobileCase.tsx

```diff
@@ -66,7 +66,7 @@ const MobileCase: React.FC<Props> = ({
       heading={capitalize(caseTypes[theCase.type])}
       onClick={onClick}
       tags={[
-        <Tag variant={tag.color} outlined disabled>
+        <Tag variant={tag.color} outlined disabled key={tag.text}>
```

Comment:
I think there are going to be non-unique keys if we do this. Consider using something else, like a uuid or a timestamp.

Reply:
It is not a problem here since we explicitly say only a single tag is rendered.

Reply:
> The key only has to be unique among its siblings, not globally unique.
https://reactjs.org/docs/reconciliation.html#keys